src/main.rs

```diff
@@ -62,7 +55,7 @@ fn run() -> Result<(), MainError> {
                 .map_err(MainError::from_dyn)?
                 .pipe(serde_json::de::from_reader::<_, NodeManifest>)
                 .map_err(MainError::from_dyn)?;
-            let name = cli.name[1].clone();
+            let name = args.script.clone();
```

Comment:
This `.clone()` looks ugly, I will do something about it later in my editor.

Reply:
the clone wasn't needed. I removed it.

Reply:
I see that the Clippy CI complains about your code multiple times. Did you set `clippy` as the "check" command for your rust-analyzer in VS Code? If you didn't, you should do it now.

Reply:
You can do so by adding `"rust-analyzer.check.command": "clippy"` to your `settings.json` in your VS Code editor.

Reply:
right, but I use neovim for development. I open vs code to check clippy but maybe I missed it. Is there a way to run it through CLI?

Reply:
I don't use neovim but it should also support rust-analyzer. There should be equivalent settings somewhere.

Reply:
I forgot to answer this question. 

> Is there a way to run it through CLI?

Just run `cargo clippy`.